src/bptl/tasks/views.py

```diff
@@ -19,3 +19,29 @@ class AddTaskMappingView(LoginRequiredMixin, CreateView):
     model = TaskMapping
     form_class = TaskMappingForm
     success_url = reverse_lazy("tasks:taskmapping-list")
+
+    def get_formset(self):
```

Comment:
django-extra-views has some useful base views to deal with formsets, please look at that next time :)

Reply:
Thanks! I'll tasks a look